internal/cloud/gcp/gcp.go

```diff
@@ -124,6 +124,7 @@ func (c *Cloud) Close() {
 }
 
 // GetLoadBalancerEndpoint returns the endpoint of the load balancer.
+// Warning: on GCP, this returns a hostname:port, not just a hostname.
```

Comment:
From our dev-docs: [endpoint: host+port](https://github.com/edgelesssys/constellation/blob/main/dev-docs/conventions.md#network), so returning hostname:port should be the expected outcome.
If this works differently for other CSPs, the warning should be moved there.

Reply:
Currently, only GCP returns a valid endpoint (host+port).
Maybe we should just fix the other CSPs to also return host+port?